src/api/python/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ target_include_directories(pycvc5
   ${CMAKE_BINARY_DIR}/src         # for cvc5_export.h
 )
 
-target_link_libraries(pycvc5 cvc5 ${PYTHON_LIBRARIES})
+target_link_libraries(pycvc5 cvc5)
```

Comment:
How pycvc5 linked against the Python library if this is removed here?

Reply:
Good question, I'm just referring to [this section](https://www.python.org/dev/peps/pep-0513/#libpythonx-y-so-1) of PEP-513 on this. It sounds like it is almost never necessary and can actively be bad in some cases. Essentially, it's not needed because once the extension module is loaded into a running interpreter, it automatically has access to all the interpreter's libpython symbols.